secretkeys/secretkeys.go

```diff
@@ -0,0 +1,39 @@
+package secretkeys
+
+import (
+	"strings"
+
+	"github.com/bitrise-io/go-utils/v2/env"
+)
+
+const (
+	// EnvKey is the shared env var key
+	EnvKey    = "BITRISE_SECRET_ENV_KEY_LIST"
```

Comment:
Can this be private?

Reply:
No, because the CLI uses this key to set the necessary env var with envman. I could have combined envman with this code but the CLI was using it directly so I thought it is not necessary and just a constant is enough.